nemo_curator/modules/semantic_dedup.py

```diff
@@ -99,9 +98,7 @@ def _mean_pooling(self, model_output, attention_mask):
 class EmbeddingCrossFitModel(HFModel):
     def __init__(self, config: EmbeddingConfig):
         self.config = config
-        super().__init__(
-            self.config.model_name_or_path, max_mem_gb=self.config.max_mem_gb
-        )
+        super().__init__(self.config.model_name_or_path)
```

Comment:
We should pass in max_mem_gb similar to how we do here by letting folks configure it automatically if not provided.  

https://github.com/NVIDIA/NeMo-Curator/blob/36fcf50cee12ccd3e85b204f7ef8c4f62c84aa51/nemo_curator/classifiers/fineweb_edu.py#L32-L37